member_organizations.yaml

```diff
@@ -46,6 +46,14 @@
     - kimwnasptd@arrikto.com
   url: https://www.arrikto.com
 
+- name: Bloomberg
+  contacts:
+    - dsun20@bloomberg.net
+    - klaban1@bloomeberg.net
+    - ifilonenko@bloomberg.net
+  url: http://www.bloomberg.com
+  hiringUrl: https://www.bloomberg.com/careers
```

Comment:
Do we really need the hiringUrl in here?

Reply:
no, let me take it off.

Reply:
thanks